libs/wire-api/src/Wire/API/Team/Permission.hs

```diff
@@ -18,6 +18,7 @@
 --
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
+{-# OPTIONS_GHC -Wno-unused-top-binds #-}
```

Comment:
No, this is because `genSingletons` generates a lot of type definitions (for promoted constructors), and we would either need to export them or use them somehow. I think the most recent version of singletons fixes this, but upgrading is slightly non-trivial. So for now we'll have to accept the warning suppression, I think.

Reply:
If you could put this explanation with a futurework tag next to the option setting, that would be great! Anyone else stumbling upon it will have an explanation right there until we get to a more recent version of the singletons library.

Reply:
Done.